utils/check-style/check-style

```diff
@@ -168,7 +168,7 @@ tests_with_query_log=( $(
 ) )
 for test_case in "${tests_with_query_log[@]}"; do
     grep -qE current_database.*currentDatabase "$test_case" || {
-        grep -qE 'current_database.*\$CLICKHOUSE_DATABASE' "$test_case"
+        grep -qE 'current_database.*CLICKHOUSE_DATABASE' "$test_case"
```

Comment:
You don't need all these changes. Every test executes in its own (random) database. Don't create a new database and instead use any table without quotes. For querying, using `WHERE database = currentDatabase() AND name ...`

Reply:
Roger! I realized I could have used for simplicity `CLICKHOUSE_DATABASE` afterwards, since I based checking the `system.tables` in `00080_show_tables_and_system_tables.sql`, which is using `CLICKHOUSE_DATABASE` and would have never gone through the current `check-style` as it is.

I'll switch to `currentDatabase()` then